backend/main.py

```
@@ -137,46 +138,86 @@ async def get_db_config(user: dict = Depends(require_auth)):
-    if conversation is None:
+    chat = database.get_chat_by_id(conversation_id)
+    if chat is None:
         raise HTTPException(status_code=404, detail="Conversation not found")
 
     # Check ownership
-    if conversation.get("user_id") != user["user_id"]:
+    if chat.get("user_id") != user["user_id"]:
         raise HTTPException(status_code=403, detail="Access denied")
 
-    success = storage.delete_conversation(conversation_id)
+    success = database.delete_chat(conversation_id)
```

Ruff (0.14.10)

Local variable `success` is assigned to but never used

Remove assignment to unused variable `success`

(F841)